daemons/gptp/common/ieee1588port.cpp

```diff
@@ -1274,17 +1276,20 @@ void IEEE1588Port::becomeSlave( bool restart_syntonization ) {
 
 	port_state = PTP_SLAVE;
 
-	/* TODO : Seems this be active here
+	/* TODO : Should this be active for non-automotive profile uses
```

Comment:
Also I agree with this comment but decided to just add the TODO so that it could be looked at in more detail when someone has a chance.